app/javascript/packs/application.js

```diff
@@ -9,12 +9,13 @@ require('channels');
 
 import 'translations/translations';
 
+import 'stylesheets/_core.scss';
 import 'initializers/';
 import 'screens/';
 
 // Uncomment to copy all static images under ../images to the output folder and reference
 // them with the image_pack_tag helper in views (e.g <%= image_pack_tag 'rails.png' %>)
 // or the `imagePath` JavaScript helper below.
-//
-// const images = require.context('../images', true)
-// const imagePath = (name) => images(name, true)
+
+const images = require.context('../images', true);
+const imagePath = (name) => images(name, true);
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/801dc79f134840871ce63769348d824626712f34/app/javascript/packs/application.js#L21"></span>‘imagePath’ is assigned a value but never used.</td>
    </tr>
  </tbody>
</table>